openslides_backend/presenter/search_users_by_name_or_email.py

```diff
@@ -67,9 +67,9 @@ def get_result(self) -> Any:
         result: Dict[str, List[Dict[str, Union[str, int]]]] = defaultdict(list)
         filter_bulk_tuples: Set[Tuple[str, str]] = set()
         for search in self.data["search"]:
-            if username := search.get("username"):
+            if username := search.get("username", "").strip():
                 filter_bulk_tuples.add(("username", username))
-            if email := search.get("email"):
+            if email := search.get("email", "").strip():
```

Comment:
Although it would make sense, we currently do not strip the email field. So either remove this here or add this in the user mixin so the stripping is consistent.

Reply:
done, stripped the email